setup.py

```diff
@@ -96,7 +96,7 @@ def get_version(version_file):
         "defusedxml",
         "dnspython",
         "docopt",
-        "mongoengine >= 0.16.3",
+        "mongoengine >= 0.16.3,<0.27.0",
```

Comment:
Can you please add a comment here reminding us why we are pinned to this version?

Reply:
Done! Updated [2797a50](https://github.com/cisagov/admiral/pull/49/commits/2797a500ee57a881ccd11f2ad9f59ad3798cbb12).

Reply:
Sorry I missed this before, but can you also please create an issue in this repo explaining this limitation and mention any potential solutions that would allow us to get to a newer version mongoengine?  After that, please add a reference to that issue in this PR description and the code comment you added in https://github.com/cisagov/admiral/pull/49/commits/2797a500ee57a881ccd11f2ad9f59ad3798cbb12.

Reply:
Sure! I linked #9 in [76fd2a6](https://github.com/cisagov/admiral/pull/49/commits/76fd2a638c3c5e52c29fde39b0e4fa64b22bbbd4).  Do you think that's appropriate (after I update the issue) or should I create a new one?

Reply:
I created #50 and reference it in [110660c](https://github.com/cisagov/admiral/pull/49/commits/110660ce001262564d8f59ddb0f68ae55c5fb68e). I also updated the reference in [5f2bb25](https://github.com/cisagov/admiral/pull/49/commits/5f2bb253a20ae13e52130956d37f5a99b1812f62).

Reply:
Perfect, that all looks great now!